mock/mock.go

```diff
@@ -923,56 +923,73 @@ func (args Arguments) Is(objects ...interface{}) bool {
 	return true
 }
 
+type outputRenderer interface{}
```

Comment:
testify supports go 1.17+ so we'd need to wait for that version to drop before cutting this over to generics